sdk/core/loader/boot.cc

```diff
@@ -1460,6 +1460,7 @@ extern "C" SchedulerEntryInfo loader_entry_point(const ImgHdr &imgHdr,
 	// invoke the exception entry point.
 	auto exportEntry = build<ExportEntry>(
 	  imgHdr.scheduler().exportTable,
+	  // XXX NWF
```

Comment:
???

Reply:
Whoops!  Sorry, that was a note to self in some of the switcher reworking that got committed accidentally.